packages/react-card/src/components/Card/useCardStyles.ts

```diff
@@ -6,21 +6,50 @@ import type { CardState } from './Card.types';
  */
 const useStyles = makeStyles({
   root: theme => ({
-    // TODO Add default styles for the root element
+    display: 'flex',
+    flexDirection: 'column',
+    overflow: 'hidden',
```

Comment:
why is an explicit `overflow: hidden` needed ?

Reply:
For scenarios where a CardPreview is are used as the top element, the image would overlay the rounded corners of the parent.